app/controllers/comments_controller.rb

```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
```

Comment:
Setting @comment.author = current_user in the create action is good for associating comments with the current user. However, ensure that the author_id parameter is not allowed through comment_params to prevent users from tampering with the authorship of comments.